<launch>
  <node name="joy_node" pkg="joy" type="joy_node" >
    <param name="dev" value="/dev/input/js1" />
  </node>

  <node name="joy_test" pkg="joy_test" type="joy_test_node" output="screen" >
    <param name="angle1" value="0" />
    <param name="angle2" value="30" />
    <param name="angle3" value="-30" />
    <param name="angle4" value="-60" />
  </node>

  <node name="arduino" pkg="rosserial_arduino" type="serial_node.py" output="screen">
     <param name="port" value="/dev/ttyACM0"/>
     <param name="capableDuty" value="0.2"/>
     <param name="Kp" value="0.25"/>
     <param name="Ki" value="0.0"/>
     <param name="Kd" value="0.0"/>
  </node>
</launch>
